include/linker/linker-tool-gcc.h

```diff
@@ -16,7 +16,11 @@
 #define ZEPHYR_INCLUDE_LINKER_LINKER_TOOL_GCC_H_
 
 #if defined(CONFIG_ARM)
+#if defined(CONFIG_BIG_ENDIAN)
+	OUTPUT_FORMAT("elf32-bigarm")
+#else
 	OUTPUT_FORMAT("elf32-littlearm", "elf32-bigarm", "elf32-littlearm")
+#endif
```

Comment:
You can make it like so:

OUTPUT_FORMAT("elf32-littlearm")

There is no need for multiple formats, because we will not support the -EB or -EL flags.

Also, you could remove duplication by doing;

#if defined(CONFIG_BIG_ENDIAN)
#define OUTPUT_FORMAT_ "elf32-bigarm"
#else
#define OUTPUT_FORMAT_ "elf32-littlearm"
#endif

OUTPUT_FORMAT(OUTPUT_FORMAT_)

Reply:
Fixed.